code/addons/actions/package.json

```diff
@@ -68,26 +68,23 @@
     "prep": "node --loader ../../../scripts/node_modules/esbuild-register/loader.js -r ../../../scripts/node_modules/esbuild-register/register.js ../../../scripts/prepare/addon-bundle.ts"
   },
   "dependencies": {
-    "@storybook/core-events": "workspace:*",
     "@storybook/global": "^5.0.0",
     "@types/uuid": "^9.0.1",
     "dequal": "^2.0.2",
     "polished": "^4.2.2",
     "uuid": "^9.0.0"
   },
   "devDependencies": {
-    "@storybook/client-logger": "workspace:*",
-    "@storybook/components": "workspace:*",
-    "@storybook/manager-api": "workspace:*",
-    "@storybook/preview-api": "workspace:*",
-    "@storybook/theming": "workspace:*",
-    "@storybook/types": "workspace:*",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-inspector": "^6.0.0",
     "telejson": "^7.2.0",
     "typescript": "^5.3.2"
   },
+  "peerDependencies": {
+    "@storybook/test": "workspace:*",
```

Comment:
I did this based on a recommendation by @kasperpeulen 

I'm totally open to changing it to a broad version range (I think we should likely do that for all packages at some point).

But I'll loop @kasperpeulen in to have a say if he sees any potential problems to broadening it.

Reply:
As far as I know, addon-actions has no dependency on `@storybook/test`.

It only has a type dependency:
<img width="852" alt="image" src="https://github.com/storybookjs/storybook/assets/1035299/4968a5e1-eb81-4a6a-bbb8-96c1b17e2d58">

Reply:
👆 That's the only reference in the actions addon, if we added `@storybook/test` as a peer dependency it would be a breaking change!

Reply:
@kasperpeulen I removed it, as per request, but it IS being used:
https://github.com/storybookjs/storybook/blob/cb5b778aac3e050899864c38b2e2faa50aa4a47e/code/addons/actions/src/loaders.ts#L4

What's the suggested solution here?

AFAICR I added this peerDep based on your recommendation.

Reply:
CI is failing now:
https://app.circleci.com/pipelines/github/storybookjs/storybook/78948/workflows/e3459292-d326-4452-82b7-beb882a980de/jobs/673389

Reply:
I removed the peerDep requirement.

It IS used in our `src` code for the package, but as @kasperpeulen mentions only importing a type.
And then this type isn't re-exported.

So -for now- it's fine to not list this, because it will not impact runtime bundle nor type bundle.
When the usage changes, (not planned) this would change.